tests/frontier/scenarios/programs/all_frontier_opcodes.py

```diff
@@ -0,0 +1,81 @@
+"""
+Define a program for scenario test that executes all frontier opcodes and entangles it's result
+"""
+import pytest
+
+from ethereum_test_tools import Bytecode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+def make_all_opcode_program() -> Bytecode:
```

Comment:
I think this could be a fixture and then it could be that we add opcodes as they become active through the forks:
```suggestion
def operation_result(sender: EOA, fork: Fork) -> Tuple[Bytecode, int]:
```
The return value would become:
```python
return (code, result)
```
where result, as of now, is `0x5831F0D814F4B8434FFDCE4DD24B00D8B7A3F67F8C316EC51A4D9EEC535AD4A`.

But then you can check for opcodes in `fork.valid_opcodes()`, and start appending code at the end (or rather before the `Op.RETURN(0, 32)`) and mutate the expected result:

```python
    result = 0x5831F0D814F4B8434FFDCE4DD24B00D8B7A3F67F8C316EC51A4D9EEC535AD4A  # As of Frontier
    if Op.PUSH0 in fork.valid_opcodes():
        code += Op.MSTORE(0, Op.SUB(Op.MLOAD(0), Op.PUSH0))
        result -= 0
    code += Op.RETURN(0, 32)
    return (code, result)
```

The `sender` parameter you can use it to predict the value of `Op.ORIGIN`.

One last thing is that you would have to move this function to either `test_scenarios.py` or a `conftest.py` file (so it would be picked up by any `test_*.py` file in the `scenarios` folder.

Reply:
I made a return parameter system with examples how it will look like with verification of 
```
address
caller
origin
selfbalance
```
opcodes

now it gets tricky as each scenario must compute this opcodes for test validation